lvgl/Cargo.toml

```diff
@@ -105,8 +110,3 @@ required-features = ["alloc", "embedded_graphics"]
 name = "sdl"
 path = "../examples/sdl.rs"
 required-features = ["alloc", "drivers"]
-
-[[example]]
-name = "memtest"
-path = "../examples/memtest.rs"
-required-features = ["lvgl_alloc"]
```

Comment:
This was an oopsie, forgot to delete earlier when I was testing stuff. Shouldn't have ever made it to master 😓